pkg/route/ingress/ingress.go

```diff
@@ -511,9 +548,17 @@ func (c *Controller) sync(key queueKey) error {
 	return utilerrors.NewAggregate(errs)
 }
 
+// validOwnerRefAPIVersions is a set of recognized API versions for the ingress
+// owner ref.
+var validOwnerRefAPIVersions = sets.NewString(
+	"networking.k8s.io/v1",
+	"networking.k8s.io/v1beta1",
+	"extensions.k8s.io/v1beta1",
```

Comment:
If IIRC, `extensions.k8s.io/v1beta1` has been removed in k8s, and was not previously allowed. Why are we allowing this apiVersion here?

Reply:
We allow it because a route might have been created with an owner reference that uses the old apiVersion.  See https://bugzilla.redhat.com/show_bug.cgi?id=1896678
 for example.